vispy/util/ptime.py

```diff
@@ -32,7 +32,7 @@ def unixTime():
     return systime.time()
 
 
-if sys.platform.startswith('win') and sys.version < '3.3':
+if sys.platform.startswith('win') and (3 > sys.version_info[0] or (3 == sys.version_info[0] and 3 > sys.version_info[1])):
```

Comment:
I guess this same as writing `sys.version_info > (3, 3)`

Reply:
It is actually the opposite (less than 3.3), but doesn't matter overall as we don't support Python <3.7 so no point in checking this anymore.